server/mvcc/backend/tx_buffer.go

```diff
@@ -88,12 +93,19 @@ func (txr *txReadBuffer) ForEach(bucketName []byte, visitor func(k, v []byte) er
 	return nil
 }
 
+// generate a new cached txReadBuffer, caller should acquire backend.readTx.RLock()
```

Comment:
"caller should acquire" -> by convention we call such methods "unsafeGenerateNew..."

Reply:
done